controllers/scenario/scenario_adapter.go

```diff
@@ -149,30 +119,8 @@ func (a *Adapter) EnsureDeletedScenarioResourcesAreCleanedUp() (controller.Opera
 		return controller.ContinueProcessing()
 	}
 
-	environmentsForScenario, err := a.loader.GetAllEnvironmentsForScenario(a.client, a.context, a.scenario)
-	if err != nil {
-		a.logger.Error(err, "Failed to find all Environments for IntegrationTestScenario")
-		return controller.RequeueWithError(err)
-	}
-	for _, testEnvironment := range *environmentsForScenario {
-		testEnvironment := testEnvironment
-		if h.IsEnvironmentEphemeral(&testEnvironment) {
-			dtc, err := a.loader.GetDeploymentTargetClaimForEnvironment(a.client, a.context, &testEnvironment)
-			if err != nil || dtc == nil {
-				a.logger.Error(err, "Failed to find deploymentTargetClaim defined in environment", "environment.Name", &testEnvironment.Name)
-				return controller.RequeueWithError(err)
-			}
-
-			err = h.CleanUpEphemeralEnvironments(a.client, &a.logger, a.context, &testEnvironment, dtc)
-			if err != nil {
-				a.logger.Error(err, "Failed to delete the Ephemeral Environment")
-				return controller.RequeueWithError(err)
-			}
-		}
-	}
-
```

Comment:
Is this removal necessary for this PR? If not, it can be handled as part of [STONEINTG-815](https://issues.redhat.com/browse/STONEINTG-815).

Reply:
Let me switch it back, you re right, it should be part of 815